src/hooks/useDailyStatistics/utils.test.tsx

```diff
@@ -155,24 +161,29 @@ describe("countTotalTransactionsAndByCategory", () => {
     ];
   });
 
-  it("should return multiple summarised transactions categories with total count and count per category and the name to be displayed on the stats page, as well as ordered by ascending order number", () => {
+  it("should return multiple summarised transactions categories with total count and count per category and the name to be displayed on the stats page, as well as ordered by ascending order number", async () => {
     expect.assertions(1);
+
     expect(
-      countTotalTransactionsAndByCategory(pastTransactions, campaignPolicy)
+      countTotalTransactionsAndByCategory(
+        pastTransactions,
+        campaignPolicy,
+        jest.fn()
+      )
     ).toStrictEqual({
       summarisedTotalCount: 4019,
       summarisedTransactionHistory: [
         {
           category: "instant-noodles",
           name: "🍜 Instant Noodles",
-          quantityText: "999 pack(s)",
+          quantityText: "999 qty",
```

Comment:
I noticed now we're just testing for `qty` while previously we explicitly tested for `pack(s)`. We should be mindful of changing test cases as there are some functionalities being tested that could be lost  with such changes.

For this particular case, is it useful to test for `pack(s)`?

Reply:
added mock